pkgs/development/libraries/kokkos/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, gitUpdater
+, cmake
+, python3
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "kokkos";
+  version = "4.1.00";
+
+  src = fetchFromGitHub {
+    owner = "kokkos";
+    repo = "kokkos";
+    rev = finalAttrs.version;
+    hash = "sha256-bPgXn1Lv+EiiKEHgTVhRFhcELUnZCphaXDlrTYq6cpY=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    python3
+  ];
+
+  cmakeFlags = [
+    "-DKokkos_ENABLE_TESTS=ON"
```

Comment:
```suggestion
    (lib.cmakeBool "Kokkos_ENABLE_TESTS" true)
```

Reply:
Interesting. This makes sense so we don't have to do do stuff like ``-DVALUE=${if x then y else z}``, we can just do ``lib.cmakeBool "VALUE" x``.